portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -9,16 +12,27 @@
 @WebServlet("/form-handler")
 public class FormHandlerServlet extends HttpServlet {
 
-  @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        // Get the value entered in the form.
+        String name = request.getParameter("name");
+        String email = request.getParameter("email");
+        String phone = request.getParameter("phone");
+        String message = request.getParameter("message");
+
 
-    // Get the value entered in the form.
-    String textValue = request.getParameter("text-input");
+        // Print the value so you can see it in the server logs.
+        System.out.println("You submitted name: " + name);
```

Comment:
Prefer one log line rather than 5. If this is a debug line for the server logs, you should keep it simple.

```
System.out.printf(
"Received name:%s \n email:%s\n phone:%s\n message:%s", name, email, phone, message);
```